src/omv/img/imlib.c

```diff
@@ -471,8 +472,66 @@ void imlib_bayer_to_rgb565(image_t *img, int w, int h, int xoffs, int yoffs, uin
             g = IM_G826(g);
             b = IM_B825(b);
             *rgbbuf++ = IM_RGB565(r, g, b);
-        }
-    }
+        } // for x
+    } else { // faster way
+        uint32_t l0,l1,l2; // 3 lines we're working with
```

Comment:
Can you add some comments on what this code is trying to do? I'm having a very hard time following it.

Reply:
added